dev/preview/ssh-proxy-command.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+VM_NAME="$(git symbolic-ref HEAD 2>&1 | awk '{ sub(/^refs\/heads\//, ""); $0 = tolower($0); gsub(/[^-a-z0-9]/, "-"); print }')"
+NAMESPACE="preview-${VM_NAME}"
+
+while getopts n:p: flag
+do
+    case "${flag}" in
+        n) NAMESPACE="${OPTARG}";;
+        p) PORT="${OPTARG}";;
+        *) ;;
+    esac
+done
+
+pkill -f "kubectl --context=harvester (.*)${PORT}:22"
+kubectl \
+    --context=harvester \
+    --kubeconfig=/home/gitpod/.kube/config \
+    -n "${NAMESPACE}" port-forward service/proxy "${PORT}:22" > /dev/null 2>&1 &
+
+# Wait for the port to be read
+sleep 5
```

Comment:
Was hoping to use `nc` here to check for the "readiness" of port $PORT in a while loop and sleep 1 for each failure. With 5 seconds it might be fragile.

Reply:
Done